common/constants.py

```diff
@@ -15,6 +15,8 @@
 VMAAS_REPOS_ENDPOINT = f"{CFG.vmaas_host}/api/vmaas/v3/repos"
 VMAAS_OS_ENDPOINT = f"{CFG.vmaas_host}/api/vmaas/v3/os/vulnerability/report"
 
+SINGLE_ORG_VMAAS_ORGANIZATION = "DEFAULT"
```

Comment:
The default org label in Foreman is `Default_Organization`.

Ideally we could have this configurable, as it is configurable in Foreman. Maybe in next iteration...

Reply:
We can improve it in the future, at this moment I don't know how